# pass $CC_BITS to link_executable to make sure the linker is called
# correctly

--- netifaces-0.10.4/setup.py.~1~	2014-05-19 04:39:39.000000000 -0700
+++ netifaces-0.10.4/setup.py	2015-02-25 11:00:31.638387652 -0800
@@ -78,7 +78,8 @@ class my_build_ext(build_ext):
                                                   output_dir=self.build_temp,
                                                   library_dirs=library_dirs,
                                                   libraries=libraries,
-                                                  debug=self.debug)
+                                                  debug=self.debug,
+                                                  extra_preargs=[os.environ["CC_BITS"]])
                     if execute:
                         abspath = os.path.abspath(os.path.join(self.build_temp,
                                                                'conftest'))
